doghouse/server/cookieman/cookieman.go

```diff
@@ -0,0 +1,109 @@
+package cookieman
+
+import (
+	"encoding/base64"
+	"net/http"
+)
+
+type Cipher interface {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported type Cipher should have comment or be unexported